### ISSUE TITLE
Add birth date input

### ISSUE THREAD

**Issue by dpikt** [LaunchPadLab/lp-components#393]
https://uxmovement.com/forms/bad-practices-on-birthdate-form-fields/

**Comment by chawes13** [comment]
+1 to this. What about an expiration date input?

**Comment by dpikt** [comment]
Like for cards?

**Comment by chawes13** [comment]
Yep! Working across a few different payment providers I've found that that field is often not included in the fields that are provided via iFrame.

**Comment by chawes13** [comment]
We have a reference implementation of an input with validations and an input mask in SC

**Comment by marchwiany** [comment]
@dpikt I want to research this a bit more. In another article from the same author (about phone numbers) he lists out some cons of splitting fields that he doesn't mention in the other article: https://uxmovement.com/forms/bad-practices-on-phone-number-form-fields/ So I think I'd want to weigh all the pros and cons and also try to find some usability test research.
